pyspecProcScripts/correlation_alignment.py

```diff
@@ -147,9 +114,9 @@ def correl_align(
     signal_keys = list(signal_pathway)
     signal_values = list(signal_pathway.values())
     ph_len = {j: psd.ndshape(s_orig)[j] for j in signal_pathway.keys()}
-    N = psd.ndshape(s_orig)[indirect_dim]
+    N = psd.ndshape(s_jk)["repeats"]
```

Comment:
N is the length of the indirect which now might be smooshed and include nScans - s_jk has smooshed dim